### ISSUE TITLE
Setting up dev enviornment did not work by following steps exactly as mentioned in the docs

### ISSUE THREAD

**Issue by ChidambaramR** [TEAMMATES/teammates#9177]
- **Commit SHA:** NA
- **Operating System:** MacOS Sierra ( 10.12.6 )
- **JDK Version:** 1.8
- **Node.js Version:** 6.4.1 (npm -v)
- **Google Cloud SDK Version:** 1.7.1.201807071456
- **Eclipse / IntelliJ Version (if applicable):** Eclipse Java EE IDE for Web Developers., 2018-09 (4.9.0)
- **Firefox / Chrome Version (if applicable):**

While setting up dev environment. I am facing two issues

- After disabling validations for javascript files in node_modules folder, I still find a lot (88) javascript problems when doing a clean build. The JS errors are NOT the import errors as mentioned in the doc. The errors are syntax related. (for ex, unexpected semi colon etc). I ended up disabling all validations (i.e unchecked the build box) and now there are no errors.

Example error,

In teammates/src/mainwebapp/dev/js/main/about.js, line 42, unexpected semi-colon

- When trying to run the local server from eclipse, it did not start. I opened an issue against google-cloud-eclipse. Please refer to it here https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/3265. The local server however ran by running the gradle commands.

I am new to web development and open source . So pardon me if these questions are moot.

**Comment by wkurniawan07** [comment]
@ChidambaramR, I presume you did not run `./gradlew createConfigs`?

**Comment by ChidambaramR** [comment]
> @ChidambaramR, I presume you did not run `./gradlew createConfigs`?

I think I ran it. Attaching my history for your reference

**Comment by wkurniawan07** [comment]
Maybe you can try refreshing the project in Eclipse? Eclipse has a not-so-good reputation when it comes to syncing files.

**Comment by ChidambaramR** [comment]
I am pretty sure I ran `./gradlew createConfigs`. I am not sure if I had not run the command would other commands like ./gradlew appEngineStart / run would work.

When I ran into the issue, it seems like the appengine-web.xml file was missing from `ls -l /Users/chidr/eclipse-workspace/.metadata/.plugins/org.eclipse.wst.server.core/tmp0/teammates/WEB-INF/` directory.

I am not sure how the sync happens between `src/main/webapp/WEB-INF/appengine-web.template.xml` and the `.plugins` eclipse directory. Probably thats the one to complain.

However, apologies for the chaos created. Did the javascript errors that I reported ring a bell to you? Currently I have all JS validations disabled.
